client/app/services/usage-graphs/usage-graphs.service.js

```diff
@@ -0,0 +1,79 @@
+/** @ngInject */
+export function UsageGraphsFactory() {
+  var service = {
+    cpuChartConfig: getCPUChartConfig,
+    memoryChartConfig: getMemoryChartConfig,
+    storageChartConfig: getStorageChartConfig,
+    convertBytestoGb: convertBytestoGb,
+  };
+
+  function getCPUChartConfig(used, total) {
```

Comment:
Seems like we could maybe make 1 method, instead of 3...passing in the units and the chartId?  Thoughts?